Scripts/RemediationScripts/Remediate-EnableSSLDBForMySQLFlexibleServer.ps1

```diff
@@ -0,0 +1,817 @@
+    
+    Write-Host $([Constants]::SingleDashLine)
+    Write-Host "[Step 2 of 3] Prepare to fetch all Azure Database for MySQL flexible server(s)..."
+    Write-Host $([Constants]::SingleDashLine)
+    
+    if (-not (Test-Path -Path $FilePath)) {
+        Write-Host "Input file:  [$($FilePath)] not found. Exiting..." -ForegroundColor $([Constants]::MessageType.Error)
+        Write-Host $([Constants]::DoubleDashLine)	
+        return
+    }
+
+    Write-Host "Fetching all Azure Database for MySQL flexible server(s) from" -NoNewline
+    Write-Host " [$($FilePath)\]..." -ForegroundColor $([Constants]::MessageType.Info)
+    Write-Host $([Constants]::SingleDashLine)
+    $DBForMySQLFlexibleServerDetails = Import-Csv -LiteralPath $FilePath
+
+    $validDBForMySQLFSDetails = $DBForMySQLFlexibleServerDetails | Where-Object { ![String]::IsNullOrWhiteSpace($_.ResourceId) -and ![String]::IsNullOrWhiteSpace($_.ResourceGroupName) -and ![String]::IsNullOrWhiteSpace($_.ResourceName) }
+
+    $totalDBForMySQLFS = $(($validDBForMySQLFSDetails | Measure-Object).Count)
+
+    if ($totalDBForMySQLFS -eq 0) {
+        Write-Host "No Azure Database for MySQL flexible server(s) found. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::DoubleDashLine)	
+        return
+    }
+
+    Write-Host "Found [$(($validDBForMySQLFSDetails|Measure-Object).Count)] Azure Database for MySQL flexible server(s)." -ForegroundColor $([Constants]::MessageType.Update)
+
+    $colsProperty = @{Expression = { $_.ResourceName }; Label = "ResourceName"; Width = 30; Alignment = "left" },
+    @{Expression = { $_.ResourceGroupName }; Label = "ResourceGroupName"; Width = 30; Alignment = "left" },
+    @{Expression = { $_.RequireSecureTransport }; Label = "RequireSecureTransport"; Width = 30; Alignment = "left" },
+    @{Expression = { $_.ResourceId }; Label = "ResourceId"; Width = 100; Alignment = "left" }
+        
+    $validDBForMySQLFSDetails | Format-Table -Property $colsProperty -Wrap
+    
+    # Back up snapshots to `%LocalApplicationData%'.
+    $backupFolderPath = "$([Environment]::GetFolderPath('LocalApplicationData'))\AzTS\Remediation\Subscriptions\$($context.Subscription.SubscriptionId.replace('-','_'))\$($(Get-Date).ToString('yyyyMMddhhmm'))\SetSSLForDBForMySQLFlexibleServer"
+
+    if (-not (Test-Path -Path $backupFolderPath)) {
+        New-Item -ItemType Directory -Path $backupFolderPath | Out-Null
+    }  
+    Write-Host $([Constants]::SingleDashLine)
+    Write-Host "[Step 3 of 3] Set SSL to previous value on all Azure Database for MySQL flexible server(s) in the Subscription..."
+    Write-Host $([Constants]::SingleDashLine)
+
+    if ( -not $Force) {        
+        Write-Host "Do you want to disable SSL for all Azure Database for MySQL flexible server(s) mentioned in the file?"  -ForegroundColor $([Constants]::MessageType.Warning)
```

Comment:
remove mentioned in the file

Reply:
If we remove this they it indicates that we are going to disable SSL for all the flexible servers in the subscription.

Reply:
Aren't you printing a table before asking this question to user?

Reply:
Removing mentioned in the file. Yes we are printing the resources